src/main/java/net/pms/newgui/NavigationShareTab.java

```diff
@@ -481,13 +483,24 @@ public void itemStateChanged(ItemEvent e) {
 		});
 
 		// Hide transcoding engine names
-		hideengines = new JCheckBox(Messages.getString("FoldTab.8"), configuration.isHideEngineNames());
-		hideengines.setToolTipText(Messages.getString("FoldTab.46"));
+		hideengines = new JCheckBox(Messages.getString("FoldTab.showEngineNames"), !configuration.isHideEngineNames());
+		hideengines.setToolTipText(Messages.getString("FoldTab.showEngineNamesToolTip"));
 		hideengines.setContentAreaFilled(false);
 		hideengines.addItemListener(new ItemListener() {
 			@Override
 			public void itemStateChanged(ItemEvent e) {
-				configuration.setHideEngineNames((e.getStateChange() == ItemEvent.SELECTED));
+				configuration.setHideEngineNames((e.getStateChange() != ItemEvent.SELECTED));
+			}
+		});
+
+		// Add subtitles information to video names
```

Comment:
For a better GUI coherence, i think that the value of the live subtitles number being downloaded (20 by default) should be displayed aligned to the left border, as the others, IMO.

Reply:
@Sami32 You are saying that if other people around me steal, I should steal too? I disagree.

Numeric fields are supposed to be right-aligned. In addition I can't change it since I've used a ```CustomJSpinner``` instead of a dumb ```JTextField``` (that is, as the name suggests, intended for *text*). That means you can adjust it with the arrows or using your mouse wheel in addition to typing in the value.

I'm not going to spent the time to fix all the mess in the GUI, but I changed the other numeric fields on the "Subtitles Settings" tab so that it hopefully looks more coherent to you 😇 

See 1452b45f6c0251b6b90a2d0fc3502a6a3f40da9b.